tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -270,7 +270,7 @@ func (c *CustomClusterProvisioningTestSuite) testProvisioningRKE2CustomCluster(c
 	require.NoError(c.T(), err)
 	assert.NotEmpty(c.T(), clusterToken)
 
-	if harden.Hardened && kubeVersion < "v1.25.0" {
+	if harden.Hardened && kubeVersion < string(provisioning.HardenedKubeVersion) {
```

Comment:
another spot that previously mentioned change would affect.

Reply:
Fixed.